tests/cypress/integration/actions_tasks_objects/issue_2661_displaying_attached_files_when_creating_task.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+context('Displaying attached files when creating a task.', () => {
+    const issueId = '2661';
+    const labelName = `Issue ${issueId}`;
+    const imagesCount = 6;
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`;
+    const width = 800;
+    const height = 800;
+    const posX = 10;
+    const posY = 10;
+    const color = 'gray';
+    const archiveName = `${imageFileName}.zip`;
+    const archivePath = `cypress/fixtures/${archiveName}`;
+    const imagesFolder = `cypress/fixtures/${imageFileName}`;
+    const directoryToArchive = imagesFolder;
+    let imageListToAttach = [];
+    for (let i = 1; i < imagesCount; i++) {
```

Comment:
i = 0

Reply:
cy.imageGenerator() generates images by adding an id starting with one. For example:
```image_issue_2661_1.png, image_issue_2661_2.png, etc.```. 
If change this value, the image with the number zero will not be found. Rewritten the loop in the form 
```for (let i = 1; i <= imagesCount; i++) {```